MapboxSearch/offline/api/api-metalava.txt

```diff
@@ -118,9 +120,11 @@ package com.mapbox.search.offline {
 
   public static final class OfflineSearchEngine.Companion {
     method public com.mapbox.search.offline.OfflineSearchEngine create(com.mapbox.search.offline.OfflineSearchEngineSettings settings);
+    method public com.mapbox.common.TilesetDescriptor createPlacesTilesetDescriptor(String dataset = "mbx-main", String version = "", String? language = null);
     method public com.mapbox.common.TilesetDescriptor createPlacesTilesetDescriptor(String dataset = "mbx-main", String version = "");
     method public com.mapbox.common.TilesetDescriptor createPlacesTilesetDescriptor(String dataset = "mbx-main");
     method public com.mapbox.common.TilesetDescriptor createPlacesTilesetDescriptor();
+    method public com.mapbox.common.TilesetDescriptor createTilesetDescriptor(String dataset = "mbx-main", String version = "", String? language = null);
     method public com.mapbox.common.TilesetDescriptor createTilesetDescriptor(String dataset = "mbx-main", String version = "");
```

Comment:
Just qurious , how does the compiler/VM know witch method to call? in case the user doesn't specified language, both method overload looks valid.

As suggestion probably language must be not nullable, because behaviour calling it with null language must be the same as calling it without language

Reply:
This file is generated from source. With that said, that is the point, the language is optional so either method should be the same implementation.